stvs/SageTV7/SageTV7.xml

```diff
@@ -67182,7 +67182,7 @@
           </Action>
           <Action Name="&quot;REM Option for: Configure Placeshifter Connection&quot;" Sym="OPUS4A-210962">
            <Conditional Name="!IsEmbeddedSystem()" Sym="OPUS4A-211270">
-            <Conditional Name="( IsServerUI() || (!IsDesktopUI() &amp;&amp; IsRemoteUI()) || (IsLinuxOS() &amp;&amp; GetServerProperty(&quot;linux/enable_nas&quot;,false)) ) &amp;&amp; IsSageTVServerEnabled()" Sym="BASE-45755">
+            <Conditional Name="IsLocalConnection() &amp;&amp; IsSageTVServerEnabled()" Sym="BASE-45755">
```

Comment:
That change looks good then. And the CLA thing I believe is because it's using a different GitHub account. The one's here lists you as CraziFuzzy...whereas on the individual commits it shows you as 'christopher piper'. I know they're both you, so I'll merge it...but it may block me when I try to do that.